### PR TITLE
New Formula: Inspectrum 0.1.1

### PR DESCRIPTION
**Pull request by tduehr**
- [Y ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ Y] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ Y] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ Y] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/inspectrum.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  depends_on "pkg-config" => :build
+  depends_on "liquid-dsp"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system "inspectrum -h"
```
Comment (MikeMcQuaid) [inline Formula/inspectrum.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/inspectrum.rb (hunk 1 above)
Comment (tduehr) [inline Formula/inspectrum.rb]: not really...

`-h` does at least initialize parts of the gui...

~~~
Usage: inspectrum [options] file
spectrum viewer

Options:
  -h, --help       Displays this help.
  -r, --rate <Hz>  Set sample rate.

Arguments:
  file             File to view.
~~~

Comment (MikeMcQuaid) [inline Formula/inspectrum.rb]: Ok, that's fine. Please do `system "#{bin}/inspectrum", "-h"`.

File: Formula/inspectrum.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  desc "Offline radio signal analyser"
+  homepage "https://github.com/miek/inspectrum"
+  url "https://github.com/miek/inspectrum/archive/v0.1.1.tar.gz"
+  sha256 "b139cd7978f294d4872a1e3e70a813f4e9600f7677da5b9f6c431b3fa7f7e03e"
+
+  depends_on "qt5"
+  depends_on "fftw"
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "liquid-dsp"
```
Comment (MikeMcQuaid) [inline Formula/inspectrum.rb]: `Error: No available formula with the name "liquid-dsp"`

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/inspectrum.rb (hunk 2 above)
Comment (tduehr) [inline Formula/inspectrum.rb]: I forgot that's a tap dependency. https://github.com/dholm/homebrew-sdr/blob/master/liquid-dsp.rb

File: Formula/inspectrum.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  desc "Offline radio signal analyser"
+  homepage "https://github.com/miek/inspectrum"
+  url "https://github.com/miek/inspectrum/archive/v0.1.1.tar.gz"
+  sha256 "b139cd7978f294d4872a1e3e70a813f4e9600f7677da5b9f6c431b3fa7f7e03e"
+
+  depends_on "qt5"
+  depends_on "fftw"
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "dholm/sdr/liquid-dsp"
```
Comment (MikeMcQuaid) [inline Formula/inspectrum.rb]: Sorry, we don't want homebrew/core formulae to depend on non-homebrew/core formulae so we'll need `liquid-dsp` to get included first. This was blocked on them making a new stable release: https://github.com/Homebrew/homebrew-core/pull/2664

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by tduehr** [comment]
Okay, I'll submit that too.

**Comment by MikeMcQuaid** [comment]
@tduehr We need them to release a new stable release first. Please discuss this with upstream before opening a PR here, thanks.

**Comment by woodruffw** [comment]
Closing due to inactivity/need for upstream release.

**Pull request closed without merging**